ci/scripts/install_conda.sh

```diff
@@ -27,19 +27,21 @@ fi
 arch=$(uname -m)
 platform=$(uname)
 installer=$1
+# SC2034 (warning): version appears unused.
+# shellcheck disable=SC2034
 version=$2
 prefix=$3
 
 download_url=https://github.com/conda-forge/miniforge/releases/latest/download/${installer^}-${platform}-${arch}.sh
```

Comment:
We can use `version` here:

```suggestion
download_url=https://github.com/conda-forge/miniforge/releases/${version}/download/${installer^}-${platform}-${arch}.sh
```